controllers/llamaCPP.cc

```diff
@@ -328,29 +330,34 @@ void llamaCPP::InferenceImpl(
   if (is_streamed) {
     LOG_INFO_REQUEST(request_id) << "Streamed, waiting for respone";
     auto state = create_inference_state(this);
+    bool is_first_token = true;
```

Comment:
Should store in state since state is a throwaway object that's automatically managed by shared ptr for one single session

Reply:
@vansangpfiev

Reply:
good point, moved it to the state, thanks